portal/src/components/dropperV2/DropperV2NewDrop.tsx

```diff
@@ -111,47 +116,49 @@ const DropperV2NewDrop = ({ address, onClose }: { address: string; onClose: () =
 
   const dropTypesIds = [20, 721, 1155, 1];
 
-  const isTokenTypeValid = dropTypesIds.includes(tokenType);
-  const isAmountValid = !isNaN(Number(tokenId));
-  const isAuthorizationTokenAddressValid = web3.utils.isAddress(authorizationTokenAddress);
-  const isTokenIdValid = !isNaN(Number(tokenId));
-  const isAuthorizationPoolIdValid = !isNaN(Number(tokenId)); //TODO Check with contract ?
-  const isTokenAddressValid = web3.utils.isAddress(tokenAddress);
-  const newMetadata = useLink({ link: uri });
+  const isTokenTypeValid = dropTypesIds.includes(state.tokenType);
+  const isAmountValid = !isNaN(Number(state.tokenId));
```

Comment:
Could we disable the amount field when it's an erc721 drop and default the amount to 1? (Should be 0, but currently contract does not allow this.)

Reply:
We could force amount to be non-negative as well.